package.json

```diff
@@ -66,21 +65,27 @@
     "@opentelemetry/semantic-conventions": "1.22.0",
     "ajv": "8.13.0",
     "commander": "12.0.0",
-    "drizzle-orm": "^0.32.2",
-    "inquirer": "^8.2.6",
     "kafkajs": "^2.2.4",
     "knex": "3.1.0",
     "koa": "2.15.0",
     "lodash": "4.17.21",
     "openapi-types": "12.1.3",
     "pg": "8.11.3",
+    "pg-protocol": "^1.6.1",
     "reflect-metadata": "^0.2.2",
     "serialize-error": "8.1.0",
     "uuid": "9.0.1",
     "winston": "3.12.0",
     "winston-transport": "4.7.0",
     "yaml": "2.4.1"
   },
+  "peerDependencies": {
```

Comment:
Why are these peer dependencies?  Might these cause problems for existing installs?

Reply:
I put them as peer dependencies because if they are not installed, `ctxt.koaContext` wouldn't work. However, I might shrink this to just koa types.